packages/connect-form/src/hooks/use-connect-form.ts

```diff
@@ -411,6 +417,32 @@ export function handleConnectionFormFieldUpdate(
         },
       };
     }
+    case 'remove-proxy-options': {
+      const proxyOptions = [
+        'proxyHost',
+        'proxyPort',
+        'proxyPassword',
+        'proxyUsername',
+      ];
+      proxyOptions.forEach((key) =>
+        updatedSearchParams.delete(key as keyof MongoClientOptions)
+      );
+      return {
+        connectionOptions: {
+          ...currentConnectionOptions,
+          connectionString: parsedConnectionStringUrl.toString(),
+        },
+      };
+    }
+    case 'remove-ssh-options': {
+      return {
+        connectionOptions: {
+          ...currentConnectionOptions,
+          sshTunnel: undefined,
+          connectionString: parsedConnectionStringUrl.toString(),
```

Comment:
no need for this, we are not changing the connection string, the previous one would already be in the object from `...currentConnectionOptions`.

```suggestion
```